src/eei.cpp

```diff
@@ -744,6 +742,18 @@ inline int64_t maxCallGas(int64_t gas) {
 
       call_message.gas = gas;
 
+      if (import->base == Name("call") || import->base == Name("callCode")) {
+        try {
+          ensureSenderBalance(call_message.value);
+        } catch (OutOfGas) {
```

Comment:
This should be a const reference to avoid cloning exception objects, e.g. `OutOfGas const&`.

Reply:
Fixed